papas/pfalgo/pfblock.py

```diff
@@ -158,6 +160,30 @@ def elements_string(self):
             count = count + 1            
         return elemdetails + "      }\n"
     
+    def short_elements_string(self):
+        ''' Construct a string descrip of each of the elements in a block:-
```

Comment:
avoid abreviations when possible: 
descrip -> description? 
pontctuation at the end :- should be .  ?